src/AndroidDebugLauncher/Launcher.cs

```diff
@@ -496,6 +521,37 @@ private void VerifySdkVersion()
             }
         }
 
+        private string VerifyGdbServerPath(string workingDirectory)
```

Comment:
nit: Might want to call this 'GetGdbServerPath' since we are doing more than just verifying it.